libminifi/test/archive-tests/FocusArchiveTests.cpp

```diff
@@ -16,24 +16,23 @@
  * limitations under the License.
  */
 
+#include <algorithm>
 #include <iostream>
 #include <map>
+#include <memory>
 #include <set>
 #include <string>
-#include <algorithm>
-#include <memory>
 #include <utility>
 
-#include "../TestBase.h"
+#include <archive.h> // NOLINT
+#include <archive_entry.h> // NOLINT
```

Comment:
why do we need NOLINT here?

End of line comments should have 2 spaces before and 1 after the `//`.

Reply:
This is incorrectly recognized as a C header. Also `NOLINT` comments do not need to be preceded by 2 spaces.